src/PackageCard.tsx

```diff
@@ -1,54 +1,51 @@
 import React, { useEffect, useState } from "react";
+import { Form } from "react-bootstrap";
 import styles from "./PackageCard.module.scss";
 
-import VersionDownloadChart from "./VersionDownloadChart";
+import VersionDownloadChart, {
+  MeasurementTransform,
+  VersionFilter,
+} from "./VersionDownloadChart";
 import chartStyles from "./VersionDownloadChart.styles";
 
 import { PackageIdentifier, packages } from "./PackageDescription";
 
 export type PackageCardProps = {
   identifier: PackageIdentifier;
-  versionFilter?: "major" | "patch" | "prerelease";
+  versionFilter?: VersionFilter;
 };
 
-const MemoVersionDownloadChart: React.FC<PackageCardProps> = React.memo(
-  ({ identifier, versionFilter }) => {
-    switch (versionFilter || "major") {
-      case "major":
-        return (
-          <VersionDownloadChart
-            identifier={identifier}
-            maxVersionsShown={8}
-            versionFilter={versionFilter}
-          />
-        );
-      case "patch":
-        return (
-          <VersionDownloadChart
-            identifier={identifier}
-            maxVersionsShown={8}
-            versionFilter={versionFilter}
-          />
-        );
-      case "prerelease":
-        return (
-          <VersionDownloadChart
-            identifier={identifier}
-            versionFilter={versionFilter}
-            maxVersionsShown={4}
-          />
-        );
-    }
+type CardChartProps = {
+  identifier: PackageIdentifier;
+  versionFilter: VersionFilter;
+  measurementTransform: MeasurementTransform;
+};
+
+const CardChart: React.FC<CardChartProps> = React.memo((props) => {
+  switch (props.versionFilter) {
+    case "major":
+      return <VersionDownloadChart {...props} maxVersionsShown={8} />;
+    case "patch":
+      return <VersionDownloadChart {...props} maxVersionsShown={8} />;
+    case "prerelease":
+      return <VersionDownloadChart {...props} maxVersionsShown={4} />;
   }
```

Comment:
Nit: Could we make a `getMaxVersionsShown()` and reduce duplication here:
```ts
  function getMaxVersionsShown(versionFilter: VersionFilter): number {
    switch (versionFilter) {
      case "major":
        return 8;
      case "patch":
        return 8;
      case "prerelease":
        return 4;
    }
  }

  ...

  const maxVersionsShown = getMaxVersionsShown(props.maxVersionsShown);
  return <VersionDownloadChart {...props} maxVersionsShown={maxVersionsShown} />;
```

Reply:
Want to keep them separate, since props might diverge in the future.